packages/react-urql/examples/1-getting-started/README.md

```diff
@@ -0,0 +1,34 @@
+## React & service worker example
```

Comment:
Minor suggestion - I would avoid adding any kind of info about the service worker if possible (I think it adds more complexity and the reason behind it is to save us having to maintain a GraphQL server)

Reply:
Done.